src/samplers/hmcda.jl

```diff
@@ -52,9 +52,6 @@ function HMCDA(n_iters::Int, n_adapt::Int, delta::Float64, lambda::Float64, spac
     _space = isa(space, Symbol) ? Set([space]) : Set(space)
     return HMCDA(n_iters, n_adapt, delta, lambda, _space, 0)
 end
-function HMCDA(n_iters::Int, n_adapt::Int, δ::Float64, λ::Float64, space::Set, gid::Int)
```

Comment:
Why was this dropped?

Reply:
This constructor is available automatically by the type definition, so this was overwriting the definition of the automatic inner constructors.